docs/src/extend/shareable-configs.md

```diff
@@ -12,33 +12,44 @@ To share your ESLint configuration, create a **shareable config**. You can publi
 
 This page explains how to create and publish a shareable config.
 
+::: tip
+This page explains how to create a shareable config using the flat config format. For the deprecated eslintrc format, [see the deprecated documentation](shareable-configs-deprecated).
+:::
+
 ## Creating a Shareable Config
 
 Shareable configs are simply npm packages that export a configuration object. To start, [create a Node.js module](https://docs.npmjs.com/getting-started/creating-node-modules) like you normally would.
```

Comment:
```suggestion
Shareable configs are simply npm packages that export a configuration object or array. To start, [create a Node.js module](https://docs.npmjs.com/getting-started/creating-node-modules) like you normally would.
```

Assuming https://github.com/eslint/eslint/pull/17827#discussion_r1435578734 was intended to be added in this file.